### PR TITLE
fix: update scheduling model

### PR DESCRIPTION
**Pull request by annechung**
## Notion ticket link

[Scheduling Table Changes](https://www.notion.so/uwblueprintexecs/Team-Hub-5b0b225bfc09415ebcfad3030ce5fd30?p=7bb2b306ae03408c91a6ce800de40647)

## Implementation description
* updating category on the scheduling table to be a string array

## Steps to test
1.

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/pages/Scheduling/index.tsx [hunk 1]
```diff
@@ -32,14 +32,15 @@ interface UseStepType {
 
 const Scheduling = () => {
   const [schedulingFormValues, setSchedulingForm] = useForm({
-    category: "",
-    quantity: 0,
+    categories: "",
```
Comment (Shehryar21) [inline frontend/src/components/pages/Scheduling/index.tsx:35]: this should be an array?

File: frontend/src/components/pages/Scheduling/types.ts [hunk 2]
```diff
@@ -1,14 +1,15 @@
 import { NavigationProps, SetForm } from "react-hooks-helper";
 
 export interface SchedulingFormProps {
-  category: string;
-  quantity: number;
+  categories: string;
```
Comment (Shehryar21) [inline frontend/src/components/pages/Scheduling/types.ts:4]: type should be `string[]`

File: backend/typescript/types.ts [hunk 3]
```diff
@@ -11,6 +11,20 @@ export enum Status {
   REJECTED = "Rejected",
 }
 
+export const Category = new Set([
```
Comment (Shehryar21) [inline backend/typescript/types.ts:14]: Can we name this `Categories`?

**Review by Shehryar21 — CHANGES_REQUESTED** [review]

File: frontend/src/components/pages/Scheduling/index.tsx (hunk 1 above)
Comment (annechung) [reply frontend/src/components/pages/Scheduling/index.tsx]: ooop yep, fixed

**Review by Shehryar21 — APPROVED** [review]

**Pull request merged**